### PR TITLE
リンクの修正を行いました

### PR DESCRIPTION
**Pull request by SekiMoemi**
・商品一覧画面のリンクを変更、各ページのリンクを修正
・注文完了画面のロゴ部分に対応するリンクを記述
・ログイン時に注文履歴のリンク(th:hrefは未作成)を作成

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java [hunk 1]
```diff
@@ -24,12 +24,12 @@ public class SecurityConfig {
 	SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 		// アクセスの制限に関する設定
 		http.authorizeHttpRequests(ahr -> ahr
-				.requestMatchers("/showItemList/**", "/registerUser/**", "/login/**", "/showItemDetail/**",
+				.requestMatchers("/", "/registerUser/**", "/login/**", "/showItemDetail/**",
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:27]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 108).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:27]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java [hunk 2]
```diff
@@ -24,12 +24,12 @@ public class SecurityConfig {
...
 						"/showShoppingCart/**", "/addShoppingCart/**", "/deleteShoppingCart/**")
 				.permitAll().anyRequest().authenticated());
 		// ログインに関する設定
 		http.formLogin(login -> login.loginPage("/login/toLogin").loginProcessingUrl("/login")
-				.failureUrl("/login/toLogin?error=true").defaultSuccessUrl("/showItemList", true)
+				.failureUrl("/login/toLogin?error=true").defaultSuccessUrl("/", true)
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:32]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 101).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:32]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Review by yamada-kensyu — APPROVED** [review]
承認します！

**Pull request merged**